src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -318,13 +319,28 @@ private void testEditQuestionAction() throws Exception {
         feedbackEditPage.waitForElementVisibility(visibilityMessage);
 
         feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackQuestionVisibilityPreview.html");
+
+        ______TS("test new question (frame) link copies custom number of recipients option");
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.selectNewQuestionType("TEXT");
+        assertTrue(feedbackEditPage.isCustomNumOfRecipientsChecked());
+        feedbackEditPage.clickDiscardChangesLink(-1);
```

Comment:
Interesting, it doesn't seem very nice or understandable from the outside.. I'm not sure if it's worth an issue since it is everywhere, cc @damithc @wkurniawan07 what do you guys think?